### PR TITLE
[webapi] Integrate w3c tests for NavigatorLanguage

### PR DESCRIPTION
**Pull request by chunywang**
Change get-navigatorlanguage-manual.html precondition refer to 
https://crosswalk-project.org/jira/browse/XWALK-2132

Impacted tests(approved): new 2, update 0, delete 0
Unit test platform: Crosswalk Project for Android 15.44.371.0
Unit test result summary: pass 2, fail 0, block 0

BUG=https://crosswalk-project.org/jira/browse/XWALK-2242

### CODE REVIEW

File: webapi/tct-extra-html5-tests/w3c/system-state-and-capabilities/the-navigator-object/get-navigatorlanguage-manual.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>NavigatorLanguage: navigator.language returns the user's preferred language</title>
+<link rel="author" title="Intel" href="http://www.intel.com">
+<link rel="help" href="https://html.spec.whatwg.org/multipage/webappapis.html#navigatorlanguage">
+<script src="../../../resources/testharness.js"></script>
+<script src="../../../resources/testharnessreport.js"></script>
+<h2>Precondition</h2>
+<p>The user agent's preferred language is set as English (United Kingdom) (en-GB).</p>
```
Comment (zqzhang) [inline webapi/tct-extra-html5-tests/w3c/system-state-and-capabilities/the-navigator-object/get-navigatorlanguage-manual.html]: Seems you changed this, why?

File: webapi/tct-extra-html5-tests/tests.full.xml [hunk 2]
```diff
@@ -7624,6 +7624,35 @@
...
+            <spec_assertion category="W3C HTML5 API Specifications" element_name="language" element_type="attribute" interface="NavigatorLanguage" section="ExtraHTML5" specification="Language preferences"/>
+            <spec_url>https://html.spec.whatwg.org/multipage/webappapis.html#navigatorlanguage</spec_url>
+            <spec_statement/>
+          </spec>
+        </specs>
+      </testcase>
+      <testcase component="W3C_HTML5 APIs/W3C_EXTRAHTML5/Systemstateandcapabilities" execution_type="manual" id="get-navigatorlanguage" priority="P2" purpose="navigator.language returns the user's preferred language" status="approved" type="compliance">
+        <description>
+          <pre_condition>
+            The user agent's preferred language is set as English (United Kingdom) (en-GB).
```
Comment (zqzhang) [inline webapi/tct-extra-html5-tests/tests.full.xml]: I guess you mean here system language for Crosswalk, right?

File: webapi/tct-extra-html5-tests/w3c/system-state-and-capabilities/the-navigator-object/get-navigatorlanguage-manual.html (hunk 1 above)
Comment (zqzhang) [inline webapi/tct-extra-html5-tests/w3c/system-state-and-capabilities/the-navigator-object/get-navigatorlanguage-manual.html]: If you have to change, please refer to https://crosswalk-project.org/jira/browse/XWALK-2132 (putting this link into commit message too):

> Set Android phone language to Espanol (Estados Unidos)
> *Expected result: window.navigator.language is es-US

**Comment by chunywang** [comment]
Android device's language setting has no English(en) to select. so i modify to 'English (United Kingdom)' which can set both on browser and device

**Pull request merged**